akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/ActorTestKitSpec.scala

```diff
@@ -13,7 +13,7 @@ import org.scalatest.Matchers
 import org.scalatest.WordSpec
 import org.scalatest.WordSpecLike
 
-class ActorTestKitSpec extends ActorTestKitWordSpec {
+class ActorTestKitSpec extends ActorTestKitScalaTestSpec with WordSpecLike {
```

Comment:
We should probably create a combined ActorTestKitScalaTestSpec with WordSpecLike but users can choose there own

Reply:
I think this is good. Are there alternatives to `Matchers` also, or is that always wanted?

Reply:
Some prefer regular assertions so perhaps this and ScalaFutures should be in our private impl as they add a lot of implicits

Reply:
it was nice to change the patience in one place, or is that in Eventually and we could keep that?

one of my goals was to not have to much internal infrastructure for our tests but use the testkit as is, so the drawback would be a bit more copy paste of mixins into each test class

Reply:
We could leave it as is then as it is fairly straight forward for users to create a similar base class for their tests